python/doc/theory/data_analysis/quantile_asymptotic_confidence_estimation.rst

```diff
@@ -0,0 +1,101 @@
+.. _quantile_asymptotic_confidence_estimation:
+
+Asymptotic quantile confidence interval based on order statistics
+-----------------------------------------------------------------
+
+We consider a random variable :math:`X` of dimension 1 and its quantile :math:`x_{\alpha}`
+of level :math:`\alpha` (:math:`\alpha \in [0, 1]`).
+We want to determine an asymptotic confidence interval of :math:`x_{\alpha}` with a confidence greater or equal to
+:math:`\beta`, using order statistics.
+
+Let :math:`(X_1, \dots, X_\sampleSize)` be some independent copies of :math:`X`.
+
+Let :math:`(X_1, \dots, X_\sampleSize)` be some independent copies of :math:`X`.
+Let :math:`X_{(k)}` be the :math:`k` -th order statistics of :math:`(X_1, \dots, X_\sampleSize)` which means that
+:math:`X_{(k)}` is the :math:`k` -th minimum of :math:`(X_1, \dots, X_\sampleSize)` for :math:`1 \leq k \leq \sampleSize`. For
+example, :math:`X_{(1)} = \min (X_1, \dots, X_\sampleSize)` is the minimum
+and :math:`X_{(\sampleSize)} = \max (X_1, \dots, X_\sampleSize)` is the maximum.
+
+Empirical quantile estimator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We first detail the empirical estimator of the quantile :math:`x_{\foralpha}`.
+We denote by :math:`\hat{F}` the empirical cumulative distribution defined by:
+
+ .. math::
+
+        \hat{F}(x) = \dfrac{1}{\sampleSize} \sum_{i=1}^\sampleSize 1_{X_i \leq x}
+
+
+Then, the empirical estimator  :math:`x_{\alpha}` is defined by:
+
+ .. math::
+
+        \hat{X}_{\alpha} = \inf \{ x, \hat{F}(x) > \alpha \} = X_{(\lceil \sampleSize\alpha \rceil)}
+
+where :math:`\lceil x \rceil` is the integer ceiling of :math:`x`.
+
+The empirical estimator is asymptoically normal (see [delmas2006]_, [garnier2008]_):
+
+ .. math::
+
+        \lim_{\sampleSize \to +\infty} \sqrt{\sampleSize}( \hat{X}_{\alpha}) - x_{\alpha}) = \cN(0, \sigma^2)
+        \quad \mbox{with}  \quad \sigma^2 = \dfrac{\alpha(1-\alpha)}{p^2(x_{\alpha})}
+
+The empirical estimator has a bias and a variance of order :math:`1/\sampleSize` (see [david1981]_, [garnier2008]_, [Motoyama2025]_). We get
```

Comment:
Please be more accurate when citing (David & Nagaraja, 1981) textbook, by focusing on a specific page number or theorem.

Reply:
I do not have this reference: it was cited like that in the article I read....